### PR TITLE
Upgrade Crowdin Integration

### PR DESCRIPTION
**Pull request by gregorydlogan**
This PR updates our Crowdin integration.  The existing integration goes away towards the end of this year.

This PR also removes some files which are not present in the codebase.  I'm unclear if we want to keep the first set, will raise a review.

Also unclear if we need to put this into `r/13.x` or not.  API sunset occurs prior to full r/13.x` support drop.

Note: The relevant secret is already set in this repo's config.  The old secret should be removed (IMO) after this PR has been merged.

### Your pull request should…

* [X] have a concise title
* [x] ~[close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists~
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] ~include migration scripts and documentation, if appropriate~
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: .crowdin.yaml [hunk 1]
```diff
@@ -7,14 +8,6 @@ files:
     source: '/modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-en_US.json'
     translation: '/modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-%locale_with_underscore%.json'
 
-  -
-    source: '/modules/engage-theodul-core/src/main/resources/ui/language/en-US.json'
-    translation: '/modules/engage-theodul-core/src/main/resources/ui/language/%locale%.json'
-
-  -
-    source: '/modules/**/src/main/resources/static/language/en-US.json'
```
Comment (gregorydlogan) [inline .crowdin.yaml:15]: I can see an argument for keeping this, although there are no *current* files matching this glob.  Should we keep it?

Comment (lkiesow) [reply .crowdin.yaml:15]: If there isn't anything, let's remove it.

**Review by lkiesow — APPROVED** [review]
Looks good.
Should this go into `r/13.x` or will 13 still work with the old method until December?

**Comment by gregorydlogan** [comment]
The very end of 13 may not have the most up to date translations if the api shutdown happens as scheduled. We discussed at the technical meeting and didn't feel this was a huge deal.

I'm good with whatever branch this ends up in 🤷

**Pull request merged**